src/services/JobRoleService.ts

```diff
@@ -0,0 +1,16 @@
+
+import axios, { AxiosResponse } from "axios";
+import { JobRoleResponse } from "../models/JobRoleResponse";
+axios.defaults.baseURL = process.env.API_URL || 'http://localhost:8080/';
+
+export const URL: string = "/api/JobRoles";
+export const getJobRoles = async (): Promise<JobRoleResponse[]> => {
+    try {
+        //Change url to AWS
+        const response: AxiosResponse = await axios.get("http://localhost:8080/api/JobRoles")
```

Comment:
use the URL from above otherwise this won't work on AWS

Reply:
Would It be better to use a GitHub secret rather than a hardcoded value?

Reply:
yes but thats what the API_URL is doing above